packages/@webex/internal-plugin-mercury/test/unit/spec/mercury.js

```diff
@@ -138,7 +138,9 @@ describe('plugin-mercury', () => {
       });
 
       it('connects to Mercury using default url', () => {
+        webex.internal.feature.updateFeature = sinon.stub();
```

Comment:
Can we `webex.internal.feature.updateFeature.restore()` the stub at the end of the test? However, we are calling the sandbox. restore but while fixing the test I've noticed it doesn't get cleared sometimes.

Reply:
I've added `sinon.restore` at the end of the test(s)

Looks like `webex.internal.feature.updateFeature.restore()` isn't a valid method. Thanks, @Shreyas281299